pkg/packagekit/updates.jsx

```diff
@@ -948,7 +948,7 @@ class OsUpdates extends React.Component {
                           ? <div className="alert alert-warning">
                                 <span className="pficon pficon-warning-triangle-o"></span>
                                 <span>
-                                    <strong>{_("Cockpit itself will be updated.")}</strong>
+                                    <strong>{_("This web interface itself will be updated.")}</strong>
```

Comment:
"Software updates affect this web console."

Reply:
To me this sounds more complicated than "This web console will be updated." ("affects"? how?), but I do agree about dropping "itself" and that we can start promoting "web console".

Reply:
Yeah, this definitely is the trickier string of the two.